RiotSwiftUI/Modules/Authentication/Common/AuthenticationModels.swift

```diff
@@ -112,11 +112,14 @@ class HomeserverAddress: NSObject {
     let brand: String?
     /// The icon field is an optional field that points to an icon representing the identity provider. If present then it must be an HTTPS URL to an image resource.
     let iconURL: String?
+    /// Dertermines if this provider uses OIDC
+    let isOIDC: Bool
     
-    init(id: String, name: String, brand: String?, iconURL: String?) {
+    init(id: String = "", name: String, brand: String?, iconURL: String?, isOIDC: Bool = false) {
```

Comment:
I'm not really a fan of the default value for id now I look at it. It feels optional now, whereas it really isn't. Would be better imo to explicitly set this when we create the mock provider with a comment that its fine as there is only one.